app/javascript/lib/sentry-logging.js

```diff
@@ -54,12 +89,17 @@ function shouldLog() {
  *  Initialize Sentry to enable logging JS errors to Sentry
  */
 export function setupSentry() {
+  if (isSuppressedEnv) {
```

Comment:
A common kind of error gets suppressed, but without per-error warnings.  This covers that.

Three code paths logged from the browser to Sentry:
1. logJSFetchExceptionToSentry (when Response object has error)
2. logJSFetchErrorToSentry (like above, but passing a built-in Error object)
3. other React JS errors

When the 3rd kind of error occurs, per-error warnings become very noisy and duplicative, much more so than the other two kinds.  The idea here is to warn the user once that such errors aren't logged to Sentry, which seems valuable and is much less clutter.

Reply:
so you're saying that for (3), we won't even log those to the console?  that seems odd that we think there are errors worth logging to sentry, but that we won't even show in the console in development?

Reply:
In general, I think a statement that prints to the console every time in development is a bad idea--it will very quickly get tuned out and scrolled out of view by other messages.

Reply:
Thanks for chatting just now.  To summarize, (3) does get logged to console as an error, but is not proceeded by a warning that it's suppressed from Sentry logging.  That's because `console.warn` messages in such cases include a duplicative stack trace, making it too noisy.  To lower noise while somewhat reusing our other Sentry logging patterns, I opted to print that message once, when the page loads.

You rightly note we don't want to set a precedent of browser-console logging a bunch of environmental defaults, which seems like the strongest argument to block.

So, per our discussion, I'll plan to use per-error console messages for (3), just like the backend, and:
1.  See if `console.warn` has a parameter to omit stack traces in such cases, and if not:
2. See if `console.info` or `console.log` would omit it, and if not:
3. Abandon that pattern reuse, and drop such messages altogether